src/VisualStudio/VisualBasic/Impl/Help/HelpKeywords.vb

```diff
@@ -1,94 +1,94 @@
+        Friend Const QueryOrderBy As String = "vb.QueryOrderBy"
+        Friend Const PreprocessorConst As String = "vb.#Const"
+        Friend Const PreprocessorIf As String = "vb.#If"
+        Friend Const Region As String = "vb.#Region"
+        Friend Const [Redim] As String = "vb.Redim"
+        Friend Const ClassConstraint As String = "vb.ClassConstraint"
+        Friend Const QuerySelect As String = "vb.QuerySelect"
+        Friend Const QuerySkip As String = "vb.QuerySkip"
+        Friend Const QuerySkipWhile As String = "vb.QuerySkipWhile"
+        Friend Const QueryTake As String = "vb.QueryTake"
+        Friend Const QueryTakeWhile As String = "vb.QueryTakeWhile"
+        Friend Const Negate As String = "vb.Negate"
+        Friend Const StructureConstraint As String = "vb.StructureConstraint"
+        Friend Const QueryWhere As String = "vb.QueryWhere"
+        Friend Const XmlLiteralCdata As String = "vb.XmlLiteralCdata"
+        Friend Const XmlLiteralComment As String = "vb.XmlLiteralComment"
+        Friend Const XmlLiteralDocument As String = "vb.XmlLiteralDocument"
+        Friend Const XmlLiteralElement As String = "vb.XmlLiteralElement"
+        Friend Const XmlEmbeddedExpression As String = "vb.XmlEmbeddedExpression"
+        Friend Const XmlLiteralProcessingInstruction As String = "vb.XmlLiteralProcessingInstruction"
+        Friend Const NewConstraint As String = "vb.NewConstraint"
+        Friend Const AnonymousKey As String = "vb.AnonymousKey"
+        Friend Const AnonymousType As String = "vb.AnonymousType"
+        Friend Const ObjectInitializer As String = "vb.ObjectInitializer"
+        Friend Const ExtensionMethods As String = "vb.ExtensionMethods"
+        Friend Const Nullable As String = "vb.Nullable"
+        Friend Const AutoProperty As String = "vb.AutoImplementedProperty"
+        Friend Const XmlDocComment As String = "vb.XmlDocComment"
+        Friend Const [Rem] As String = "vb.Rem"
+        Friend Const QueryFromIn As String = "vb.QueryFromIn"
+        Friend Const QueryFromLet As String = "vb.QueryFromLet"
+        Friend Const XmlPropertyChildAxis As String = "vb.XmlPropertyChildAxis"
+        Friend Const XmlPropertyDescendantsAxis As String = "vb.XmlPropertyDescendantsAxis"
+        Friend Const XmlPropertyAttributeAxis As String = "vb.XmlPropertyAttributeAxis"
+        Friend Const XmlPropertyExtensionValue As String = "vb.XmlPropertyExtensionValue"
+        Friend Const MyNamespaceKeyword As String = "vb.My"
+        Friend Const CollectionInitializer As String = "vb.CollectionInitializer"
+        Friend Const VarianceIn As String = "vb.VarianceIn"
+        Friend Const VarianceOut As String = "vb.VarianceOut"
+        Friend Const TaskListUserComments As String = "vbc0"
+        Friend Const Main As String = "vb.Main"
+        Friend Const Async As String = "vb.Async"
+        Friend Const Iterator As String = "vb.Iterator"
+        Friend Const Await As String = "vb.Await"
```

Comment:
_Side Note:_ The colour of `Await` and `Yield` are incorrect, should be black in this context.